docs/UserGuide.md

```diff
@@ -145,22 +145,30 @@ Help is also shown if you enter an incorrect command e.g. `abcd`
 
 Adds a floating task to TaskTracker.
 
-    add "FLOATING_TASK_NAME" [PRIORITY]
+    task "FLOATING_TASK_NAME" [PRIORITY]
```

Comment:
Yeah, we could leave it like this for now.

I just implemented the add event command in #37, and I think the syntax is decent enough.